Makefile

```diff
@@ -1,6 +1,7 @@
 DESTDIR :=
-BINDIR := /usr/local/bin
-SYSCONFDIR := /etc
+PREFIX := /usr/local
+BINDIR := $(PREFIX)/bin
```

Comment:
I agree - `/usr/local/` is more trouble for us, but less surprising for other users, so the users win ;)